src/client/pages/Parkeren/useParkerenData.hook.tsx

```diff
@@ -18,7 +18,7 @@ export const PARKEER_CASE_TYPES: Set<DecosCaseType> = new Set([
   CaseType.TouringcarJaarontheffing,
 ]);
 
-function getVergunningenFromThemaVergunningen(
+function getParkeervergunningen(
```

Comment:
We hebben deze functie hernoemd omdat we binnenkort ook échte parkeervergunningen, uit Mijn Parkeren, krijgen.

```suggestion
function getVergunningenFromThemaVergunningen(
```